.github/workflows/merge-flow.yaml

```
@@ -4,7 +4,7 @@ on:
     inputs:
       node-version:
         description: node version
-        default: '16'
+        default: '22'
```

```suggestion
        default: 'lts/Jod'
```
or `22.15.0`, whichever makes more sense 🙂

---

why not `22`?
I'd like to keep it aligned with upstream prometheus which requires `>=22`

---

Actually, do we not want to keep this the same as https://github.com/openshift/release/pull/64170/files#diff-a296e21abde1dcc84c38077a45bbf1c0ca9fb1278d62fc7fca9aae5b6b14085cR52?